### PR TITLE
Dashboard overview page

### PR DESCRIPTION
**Pull request by szymanskir**

### CODE REVIEW

File: R/mod_comments_table.R [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+    
+#' @rdname mod_comments_table
+#' @export
+#' @keywords internal
+#' 
+#' @import dplyr
+#' @import stringr
+    
+mod_comments_table_server <- function(input, output, session, comments_promise) {
+  ns <- session$ns
```
Comment (ramusz1) [inline R/mod_comments_table.R:44]: unused variable

File: R/mod_general_overview_page.R [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+#' 
+#' 
+#' @importFrom future future value
+#' @importFrom hackeRnews get_comments get_top_stories
+#' @importFrom rvest html_text
+#' @importFrom xml2 read_html
+#' 
+    
+mod_general_overview_page_server <- function(input, output, session) {
+  ns <- session$ns
```
Comment (ramusz1) [inline R/mod_general_overview_page.R:67]: unused variable

File: R/mod_sentiment_distribution_plot.R [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+    
+# Module Server
+    
+#' @rdname mod_sentiment_distribution_plot
+#' @export
+#' @keywords internal
+#' @import ggplot2
+#' 
+mod_sentiment_distribution_plot_server <- function(input, output, session, comments_promise) {
+  ns <- session$ns
```
Comment (ramusz1) [inline R/mod_sentiment_distribution_plot.R:41]: unused variable

File: R/mod_stories_table.R [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+# Module Server
+    
+#' @rdname mod_stories_table
+#' @export
+#' @keywords internal
+#' 
+#' @import promises
+    
+mod_stories_table_server <- function(input, output, session, stories_promise) {
+  ns <- session$ns
```
Comment (ramusz1) [inline R/mod_stories_table.R:43]: unused variable

File: R/mod_wordcloud_plot.R [hunk 5]
```diff
@@ -0,0 +1,64 @@
...
+# Module Server
+    
+#' @rdname mod_wordcloud_plot
+#' @export
+#' @keywords internal
+#' @import ggplot2
+#' @import ggwordcloud
+    
+mod_wordcloud_plot_server <- function(input, output, session, comments_promise) {
+  ns <- session$ns
```
Comment (ramusz1) [inline R/mod_wordcloud_plot.R:42]: unused variable

**Review by ramusz1 — COMMENTED** [review]
Please remove unused variables from your code

File: R/globals.R [hunk 6]
```diff
@@ -0,0 +1 @@
+DATA_FETCHING_INTERVAL <- 60 * 3600 * 1000
```
Comment (ramusz1) [inline R/globals.R]: This seems too big, don't you mean 60 * 60 * 1000?

File: R/data_manipulation.R [hunk 7]
```diff
@@ -0,0 +1,89 @@
...
+#' @import dplyr
+#' @import stringr
+#' @import textdata
+#' @import tidytext
+calculate_sentiment <- function(sentences) {
+  data(stop_words, package = "tidytext")
+  sentiment_index <- get_sentiments("afinn")
+  
+  calculate_single_sentence_sentiment <- function(sentence) {
+    words <- sentence %>%
```
Comment (magierska) [inline R/data_manipulation.R:18]: This could be moved to new method, because it is also used in 49 line.

File: inst/app/www/styles.css [hunk 8]
```diff
@@ -0,0 +1,33 @@
...
+  background-color: var(--primary-color);
+}
+
+.box.box-solid.box-primary>.box-header {
+  color: black;
+  background: var(--primary-color);
+}
+.box.box-solid.box-primary, .box.box-primary {
+  border-color: var(--primary-color);
+  border-left-color: var(--primary-color);
```
Comment (magierska) [inline inst/app/www/styles.css]: There is no need to set color of left, right and top border.

File: R/globals.R (hunk 6 above)
Comment (szymanskir) [reply R/globals.R]: Good catch. Fixed

File: R/mod_comments_table.R (hunk 1 above)
Comment (szymanskir) [reply R/mod_comments_table.R:44]: As far as I understand this is needed for the server in order to put UI elements in the proper namespace ;) https://shiny.rstudio.com/articles/modules.html

File: R/data_manipulation.R (hunk 7 above)
Comment (szymanskir) [reply R/data_manipulation.R:18]: Good point. Fixed.

File: inst/app/www/styles.css (hunk 8 above)
Comment (szymanskir) [reply inst/app/www/styles.css]: Cool, removed ;)

**Review by ramusz1 — APPROVED** [review]

**Pull request merged**